app/controllers/support_interface/application_forms/applicant_details_controller.rb

```diff
@@ -0,0 +1,35 @@
+module SupportInterface
+  module ApplicationForms
+    class ApplicantDetailsController < SupportInterfaceController
```

Comment:
Thoughts behind "Applicant"? Perhaps otherwise `CandidateDetailsController` or `EditApplicationFormController` would be more in line with the other code.

Reply:
I followed the UI here. No particularly strong feelings though.